### PR TITLE
os: add grub.cfg ignition example

### PR DESCRIPTION
**Pull request by kbrwn**
Customer recently asked how to edit the grub.cfg via userdata. Example was confirmed by derek gonyeo.

### CODE REVIEW

File: os/other-settings.md [hunk 1]
```diff
@@ -82,6 +82,25 @@ Further details can be found in the systemd man pages:
 
 The Container Linux bootloader parses the configuration file `/usr/share/oem/grub.cfg`, where custom kernel boot options may be set.
 
+The `/usr/share/oem/grub.cfg` file can be configured with ignition. Note that ignition runs after grub. Therefore, the grub configuration won't take effect until the next reboot of the node.
```
Comment (bgilbert) [inline os/other-settings.md]: `Ignition`, capitalized.  Might as well spell it `GRUB` also.

File: os/other-settings.md [hunk 2]
```diff
@@ -82,6 +82,25 @@ Further details can be found in the systemd man pages:
...
+        device: "/dev/disk/by-label/OEM"
+        format: "ext4"
+  files:
+    - filesystem: "OEM"
+      path: "/grub.cfg"
+      mode: 0644
+      append: true
+      contents:
+        inline: |
+          set linux_append="coreos.autologin=tty1"
```
Comment (bgilbert) [inline os/other-settings.md]: This should be `set linux_append="$linux_append coreos.autologin=tty1"`.  Some OEMs already set that variable.

File: os/other-settings.md [hunk 3]
```diff
@@ -82,6 +82,25 @@ Further details can be found in the systemd man pages:
...
+```
+
```
Comment (bgilbert) [inline os/other-settings.md]: I think this addition should all be in the next subsection?

**Comment by kbrwn** [comment]
closing in favor of: https://github.com/coreos/docs/pull/1241

**Pull request closed without merging**